### DISCUSSION TITLE
Unable to start the container

### DISCUSSION THREAD

**Discussion by just5ky** [kereis/traefik-certs-dumper#47] [Q&A]
Trying to compose it on Raspberry Pi, running into an error.
when  I checked the logs in Porttrainer, its showing this

> standard_init_linux.go:228: exec user process caused: exec format error  

Here is the compose snippet
certdumper:
    container_name: traefik_certdumper
    image: humenius/traefik-certs-dumper:latest
    network_mode: none
    security_opt:
      - no-new-privileges:true
    # command: --restart-containers container1,container2,container3
    volumes:
      - $DOCKERDIR/traefik2/acme:/traefik:ro
      - $DOCKERDIR/traefik2/certs:/output:rw
      ##- /var/run/docker.sock:/var/run/docker.sock:ro # Only needed if restarting containers (use Docker Socket Proxy instead)
    environment:
      DOMAIN: $DOMAINNAME

**Comment by kereis** [comment]
Hey,

I'm currently not sure if Docker automatically uses the correct architecture for your Raspberry Pi. Maybe try to set the image tag to an according architecture explicitly? (for instance: `arm32v7-alpine`; I don't know what Raspberry Pi model you're using)

**Reply by kereis** [reply]
I close this discussion as there weren't any replies anymore.
